packages/genesys-spark-components/src/components/stable/gux-form-field/components/gux-form-field-color/gux-form-field-color.tsx

```diff
@@ -145,6 +146,7 @@ export class GuxFormFieldColor {
                 'gux-input-container': true,
                 'gux-disabled': this.disabled
               }}
+              onClick={() => focusInputElement(this.input)}
```

Comment:
When I click the padding here I get a focus ring on the input but the color selection popover doesn't appear. Is that expected?

Reply:
I removed this as its not possible to get the color picker dialog to show by just using `.focus()`